components/Valuation/Graph.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { useEffect, useState, useRef } from "react";
+import { Metaverse } from "../../lib/enums";
+import { createChart, UTCTimestamp } from "lightweight-charts";
+import { getValuationDailyData } from "../../lib/FirebaseUtilities";
+
+const Graph = ({ metaverse }: Metaverse) => {
```

Comment:
To specify that `metaverse` is of type `Metaverse` in one line then you would have to write it like this
`const Graph = ({ metaverse }: {metaverse: Metaverse}) => {`

Reply:
Also since we are gonna be using lots of graphs on the App always try to be specific with Component Names. Try and change this name into something that is more unique to this component